ui/dashboards/src/components/GridLayout/GridLayout.tsx

```diff
@@ -49,6 +52,18 @@ export function GridLayout(props: GridLayoutProps) {
     }
   };
 
+  const handleWidthChange = (
```

Comment:
Could you include a comment with a link to the relevant react-grid-layout code either here or on `calculateGridItemWidth` for reference for future us. In theory, if they change underlying calculations, something here could get out-of-sync, and it'd be nice to have a quick reference to the source-of-truth. 

https://github.com/react-grid-layout/react-grid-layout/blob/master/lib/calculateUtils.js#L14-L35